metaflow/plugins/kubernetes/kubernetes.py

```diff
@@ -710,12 +710,23 @@ def wait_for_launch(job):
         wait_for_launch(self._job)
 
         # 2) Tail logs until the job has finished
+        self._output_final_logs = False
```

Comment:
yup this should work too, and it reduces reliance on the other status internals. I'll make the changes

Reply:
scratch that, it won't work without larger changes as the `tail_logs` `has_log_updates=` callable is simply used in a while loop, which won't exhaust the decorator. This will instead get stuck tailing logs forever due to the last `True` being returned always for a finished task.